client/tracks/index.js

```diff
@@ -62,6 +62,8 @@ const events = {
 	CONNECT_ACCOUNT_KYC_MODAL_OPENED: 'wcpay_connect_account_kyc_modal_opened',
 	DEPOSITS_ROW_CLICK: 'wcpay_deposits_row_click',
 	DEPOSITS_DOWNLOAD_CSV_CLICK: 'wcpay_deposits_download',
+	OVERVIEW_BALANCE_CURRENCY_CLICK:
+		'wcpay_overview_balances_currency_tab_click',
```

Comment:
Nit/minor suggestion: use `OVERVIEW_BALANCES` naming to remain consistent with event name (`overview_balances`).
```suggestion
	OVERVIEW_BALANCES_CURRENCY_CLICK:
		'wcpay_overview_balances_currency_tab_click',
```